scripts/dog.js

```diff
@@ -0,0 +1,19 @@
+// Description
+//   Like !cat, but for dog people (and with less trauma for 2310 students
```

Comment:
forgot the ')' at the end :^)

Reply:
also, could you provide a screenshot of the dog printout? seeing as though its a visual script, it'd be nice to see what it looks like for whatever reason :)

Reply:
<img width="343" alt="screen shot 2017-11-01 at 8 04 54 pm" src="https://user-images.githubusercontent.com/9084563/32270014-fcfaf26e-bf3f-11e7-86ec-b8067364dcea.png">